composer.json

```diff
@@ -238,7 +235,8 @@
                 "Download link returns 406 with Drupal ^8.5.0 @see https://www.drupal.org/project/views_data_export/issues/2951185": "https://www.drupal.org/files/issues/2951185-2.patch"
             },
             "drush/drush": {
-                "When an update fails, updatedb is still exiting with 0. @see https://github.com/drush-ops/drush/pull/3673": "https://patch-diff.githubusercontent.com/raw/drush-ops/drush/pull/3673.patch"
+                "When an update fails, updatedb is still exiting with 0. @see https://github.com/drush-ops/drush/pull/3673": "https://patch-diff.githubusercontent.com/raw/drush-ops/drush/pull/3673.patch",
+                "Call services that need destruction at the end of request": "https://github.com/drush-ops/drush/commit/346f165cb160c554293dc4839bcba28e4368051d.diff"
```

Comment:
This commit has in the meantime been merged in to Drush, we can remove it if we use version 9.5.0 as the minimum version.